### DISCUSSION TITLE
I need help with a promises to alter rides and vendors with a staffID to for Create and Delete

### DISCUSSION THREAD

**Discussion by JuanDavila1101** [nss-evening-cohort-14/evening-client-side#201] [Help Me!] [answered]
I need help the merging getting the promise for the create and and delete for the staff

I am able to create but is not doing it all in a promise and the delete will need to be combined in one promise.
I need to merge the creates together and create one delete for both rides and vendors in one promise 

https://github.com/nss-evening-cohort-14/whalom-park-dashboard-whalom-park-dashboard/blob/jidImproveCreateStaff/src/javascripts/helpers/data/staffData/staffOnRideAndVendor.js

**Comment by ajawashington** [comment, marked as answer]
@JuanDavila1101 was being a total rockstar and going beyond MVP. This implementation was not a requirement and a misunderstanding between PO and team! 🌵
